src/components/HomeLeftContainer.vue

```diff
@@ -1,55 +1,92 @@
 <script setup lang="ts">
+import LevelDialog from "../components/LevelDialog.vue";
+import { User } from "../views/HomeView.vue";
```

Comment:
[型のimportの仕方](https://reffect.co.jp/vue/vue3-composition#Props)
```suggestion
import type { User } from "../views/HomeView.vue";
```